python/tests/test_extractor.py

```diff
@@ -0,0 +1,31 @@
+from io import StringIO
+import pytest
+from energuide import extractor
+
+
+@pytest.fixture
+def csv_string():
+    return '''EVAL_ID,EVAL_TYPE,HOUSEREGION,WEATHERLOC,ENTRYBY,CLIENTCITY,CLIENTPCODE,CLIENTNAME,TELEPHONE,\
+MAIL_CITY,MAIL_REGION,MAIL_PCODE,TAXNUMBER,RAW_XML\n123,D,Ontario,somewhere,Fred Johnson,Toronto,M5E 1W5,John \
+Fredson,999 999 9999,Toronto,Ontario,M5E 1W5,999999999999,<tag>thing</tag>'''
+
+
+def test_extract(csv_string: str):
```

Comment:
This test should test three cases: the minimum set of fields, the minimum set + some other fields (`foo`, `bar`, etc), and a set of fields that does not include the minimum set.

You can use a parameterized fixture to test both of the first two cases pass.

Reply:
By "this test" I mean "this test module". Probably two separate test functions.